joomla/joomla-cms
# [4.0] Media manager compiled using rollup

Pull Request for Issue # .

### Summary of Changes
The main course here is the remove of webpack and compiling Media Manager with Rollup, but there are couple more:
- drop the dependency `FileSaver`, we can do that in few lines: 
![Screenshot 2021-02-18 at 15 31 14](https://user-images.githubusercontent.com/3889375/108371763-5e244800-71fe-11eb-92ac-31b281f89bdb.png)
- drop the `path` dependency (the reason is that it is commonJS and ruins the whole package) for own implementation (based on the actual code of NODEJS)
- Moved the scss files in the build/media_source/com_media where is the appropriate place
- drop webpack with all the related dependencies
- Move the javascript tests to it's own package.json
- Move the `selenium-standalone` dependency directly to done.yml

### Testing Instructions
On a clean locally git cloned repo run `npm install` (or `npm ci`)
Check that the media manager has no bugs (create a folder, delete it, rename a folder, upload a file, rename a file, delete a file, download a file, edit a file)


### Actual result BEFORE applying this Pull Request


### Expected result AFTER applying this Pull Request


### Documentation Changes Required
